src/lib/components/Modal/Modal.jsx

```diff
@@ -243,52 +142,42 @@ export class Modal extends React.Component {
 }
 
 Modal.defaultProps = {
-  actions: [],
   autoFocus: true,
```

Comment:
I would use `autofocus`, I think it is one word: https://www.merriam-webster.com/dictionary/autofocus

Reply:
`autoFocus` is prop defined on all HTML input element in React with `F`, see https://reactjs.org/docs/dom-elements.html#all-supported-html-attributes

Reply:
I was looking at https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/autofocus

Apparently I was looking at the wrong doc page.